src/components/common/applicant-info-nav.vue

```diff
@@ -9,58 +9,69 @@
     </v-btn>
     <v-btn x-large
            @click="nextAction()"
-           :loading="loading"
+           :loading="getIsLoadingSubmission"
            id="submit-continue-btn">
       {{ nextText }}
     </v-btn>
   </v-col>
 </template>
 
 <script lang="ts">
-import { Component, Emit } from 'vue-property-decorator'
-import newRequestModule from '@/store/new-request-module'
-import { NameRequestMixin } from '@/mixins'
+import { Component, Emit, Vue } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { SubmissionTypeT } from '@/interfaces'
+import { ActionBindingIF } from '@/interfaces/store-interfaces'
 
 @Component({})
-export default class ApplicantInfoNav extends NameRequestMixin {
-  @Emit('nextAction')
-  private nextAction () : void {}
+export default class ApplicantInfoNav extends Vue {
```

Comment:
This is so so so so so so much better than what was there before!

I'm still sick to my stomach that there is so much left to clean up...

Reply:
But we'll get there. I'm confident that what you're doing here will solve 80% of the issues, maybe more.